src/one_dragon/base/operation/application/application_factory_manager.py

```
@@ -0,0 +1,468 @@
+        Raises:
+            ImportError: 模块导入失败
+            Other exceptions: 工厂加载或实例化时的其他错误
+        """
+        results: list[tuple[ApplicationFactory, bool, PluginInfo | None]] = []
+
+        # 确定插件包目录和模块名
+        plugin_pkg_dir = factory_file.parent
+        plugins_dir = plugin_pkg_dir.parent
+
+        if source == PluginSource.BUILTIN:
+            # 内置应用：使用从 src 开始的完整模块路径
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)